tuteria_application_test/users/serializers.py

```diff
@@ -0,0 +1,25 @@
+"""Serializers for API."""
+
+from rest_framework import serializers
+
+from .models import User, Booking
+
+class UserSerializer(serializers.ModelSerializer):
+    """Serializer for the User model."""
+
+    booking_order = serializers.SerializerMethodField()
+    transaction_total = serializers.SerializerMethodField()
+
+    def get_booking_order(self, user):
+        return [order.order for order in user.orders.all()]
+
+    def get_transaction_total(self, user):
+        total = [trans.total for trans in user.wallet.transactions.all()]
```

Comment:
the `get_transaction_total` query would cause a lot of problems if there are many users. Can you think of a solution to this problem with what you have already implemented? What change do you need to do to ensure that when fetching multiple users, there are no n+1 queries. (hint: rework your implementation of the `get_transaction_total` method